src/main/java/uk/gov/hmcts/reform/sscs/config/CitizenCcdClient.java

```diff
@@ -45,5 +49,49 @@ public CaseDetails submitForCitizen(IdamTokens idamTokens, CaseDataContent caseD
                 caseDataContent
         );
     }
+
+    List<CaseDetails> searchForCitizen(IdamTokens idamTokens) {
+        List<CaseDetails> caseDetailsList =  coreCaseDataApi.searchForCitizen(
+                idamTokens.getIdamOauth2Token(),
+                idamTokens.getServiceAuthorization(),
+                idamTokens.getUserId(),
+                ccdRequestDetails.getJurisdictionId(),
+                ccdRequestDetails.getCaseTypeId(),
+                new HashMap<>()
+        );
```

Comment:
Ideally it would be good to filter on draft appeal cases, but it's difficult to tell what values are allowed in the Map for this api call.

Reply:
no need, the search will return only cases created by that specific user. And since the user is a citizen it can only create drafts, so the search will return only drafts. Actually, only one single draft.